### DISCUSSION TITLE
Anchor in an admonition ?

### DISCUSSION THREAD

**Discussion by glassus** [squidfunk/mkdocs-material#7217] [Q&A]
Hi,
I'm a teacher and my exercises sheets are a bunch of admonitions (1 exercise = 1 admonition)
I would like to be able to link, from another page, to a specific exercise.
I can do it if I create a section (eg : ### Exercise 7) but I'm wondering if it's possible to link directly to an admonition.
Thanks,
Gilles

**Comment by alexvoss** [comment]
The [standard admonition syntax from Python Markdown](https://python-markdown.github.io/extensions/admonition/) does not allow attaching a `name` attribute to the HTML that gets generated. I am not sure if it is possible to use [attribute lists](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists).

There is an alternative [block syntax for admonitions in the Python Markdown Extensions](https://facelessuser.github.io/pymdown-extensions/extensions/blocks/plugins/admonition/), though. You would need to replace the configuration of admonitions in `mkdocs.yml`:

```yaml
markdown_extensions:
  - pymdownx.blocks.admonition 
  - pymdownx.superfences
```

Officially, mixing both types of admonitions is not supported, though I believe it works in practice. No guarantees for the future, though.

In your documents, using the block syntax would look like this:

```markdown
/// note | an example
    attrs: {name: test-example}
Hello there!
///
```

Note that the content does not need to be indented because the block syntax uses delimiters. 

Perhaps someone else can point out another path to do this.
